pos_hide_cost_price_and_margin/static/src/xml/pos_margin.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+Copyright (C) 2022 - Today: camptocamp (https://www.camptocamp.com)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<templates id="template" xml:space="preserve">
+
+    <t t-inherit="point_of_sale.ProductInfoPopup" t-inherit-mode="extension">
+        <xpath expr="//table/tr/td/t[@t-esc='costCurrency']/../.." position="replace" />
```

Comment:
This can be an issue if another module is trying to extend based on this field.
It should be hidden, not removed

Reply:
can't we set a priority on qweb templates, similar to what we do on views, to ensure that this replacement is done in the end?